build_artifacts/v2/v2.2/v2.2.0/dirs/etc/inference-server/tornado_server/sync_server.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+import asyncio
+import logging
+import tornado.web
+from utils.environment import Environment
+from utils.exception import SyncInvocationsException
+from utils.logger import SAGEMAKER_DISTRIBUTION_INFERENCE_LOGGER
+
+logger = logging.getLogger(SAGEMAKER_DISTRIBUTION_INFERENCE_LOGGER)
+
+
+class InvocationsHandler(tornado.web.RequestHandler):
+    def initialize(self, handler: callable, environment: Environment):
+        self._handler = handler
+        self._environment = environment
+
+    def post(self):
+        try:
+            self.write(self._handler(self.request))
```

Comment:
we also need to think about wrapping synchronous call into async coroutine to get some performance gains for customer.

Reply:
ack, by this are u referring to:
1. `async def post` or
2. `self.write(await self._handler(self.request))`